migrations.py

```diff
@@ -29,3 +29,14 @@ async def m001_initial(db):
         );
     """
     )
+
+
+async def m002_add_user_attrs_column(db):
+    """
+    Initial users table.
+    """
+    await db.execute(
+        """
+        ALTER TABLE usermanager.users ADD COLUMN attrs JSON
```

Comment:
is JSON i actually db type? never saw this? does this work for postgres adn sqllite? and why not just TEXT?

Reply:
yes it is, tried it out with sqlite aswell.
It's better than text because it allows for querying against json attributes with the ->> operator.